src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -446,8 +446,8 @@ namespace WindowEquivalentLayer {
         if (CFSHasControlledShade(CFS(EQLNum)) > 0) CFS(EQLNum).ISControlled = true; // is controlled
     }
 
-    void CalcEQLWindowUvalue(CFSTY const &FS, // CFS to be calculated
-                             Real64 &UNFRC    // NFRC U-factor, W/m2-K
+    void CalcEQLWindowUvalue(CFSTY &FS,    // CFS to be calculated
```

Comment:
Why can't this be const?  I don't see changes here that would restrict that away, but maybe I'm missing something.  Please clarify.